iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -27,12 +27,10 @@ enum ConversationUIState {
 class ConversationListViewController: FullScreenViewController,
                                       UICollectionViewDelegate,
                                       UICollectionViewDelegateFlowLayout,
-                                      SwipeableInputAccessoryViewDelegate, ActiveConversationable {
```

Comment:
The Convo List and Thread VCs are no longer direct delegates of the swipe accessory. Instead there is a separate delegate object.